### PR TITLE
Fix: remove old evonodes and other fixes

### PR DESCRIPTION
**Pull request by HashEngineering**

### CODE REVIEW

File: dash-sdk-java/src/test/java/org/dashj/platform/sdk/ValueTest.java [hunk 1]
```diff
@@ -164,7 +164,7 @@ public static <T> void testArrayValue(byte[] value, PlatformValue.Tag tag, Const
     @Test
     public void createPlatformValuePrimitiveTest() {
         testValue(true, PlatformValue.Tag.Bool, () -> new PlatformValue(true), PlatformValue::getBool);
-        BigInteger i128Value = BigInteger.valueOf(Long.MAX_VALUE).multiply(BigInteger.TWO);
+        BigInteger i128Value = BigInteger.valueOf(Long.MAX_VALUE).multiply(BigInteger.ONE);
```
Comment (HashEngineering) [inline dash-sdk-java/src/test/java/org/dashj/platform/sdk/ValueTest.java:167]: Android doesn't have `BigInteger.TWO`

File: dpp/src/main/java/org/dashj/platform/dashpay/BlockchainIdentity.kt [hunk 2]
```diff
@@ -2006,16 +2006,16 @@ class BlockchainIdentity {
             val xpub = decryptExtendedPublicKey(
                 contactRequest.encryptedPublicKey,
                 contactIdentity,
-                otherContactKeyIndex = contactRequest.recipientKeyIndex, // other
-                myKeyIndex = contactRequest.senderKeyIndex, // mine
+                otherContactKeyIndex = contactRequest.senderKeyIndex, // other
+                myKeyIndex = contactRequest.recipientKeyIndex, // mine
```
Comment (HashEngineering) [inline dpp/src/main/java/org/dashj/platform/dashpay/BlockchainIdentity.kt:2010]: These changes were to undo some changes from a few weeks ago that had sender and receiver backwards

File: tools/src/main/kotlin/org/dashj/platform/tools/DashPayWallet.kt [hunk 3]
```diff
@@ -158,7 +158,7 @@ class DashPayWallet(val blockchainIdentity: BlockchainIdentity, val peerGroup: P
                         if (encryptionKey == null && wallet.isEncrypted) {
                             encryptionKey = wallet.keyCrypter!!.deriveKey(password)
                         }
-                        blockchainIdentity.addContactPaymentKeyChain(contactIdentity!!, it, encryptionKey)
+                        blockchainIdentity.addPaymentKeyChainToContact(contactIdentity!!, ContactRequest(it), encryptionKey)
```
Comment (HashEngineering) [inline tools/src/main/kotlin/org/dashj/platform/tools/DashPayWallet.kt:161]: function was renamed to make its purpose more clear.

**Review by Syn-McJ — APPROVED** [review]
Looks good

**Pull request merged**